spec/models/sitemap_spec.rb

```diff
@@ -1,43 +1,41 @@
 require 'spec_helper'
 
 describe Sitemap do
-  pending('until the sitemapindexer gem file is updated for rails 5 ') do
```

Comment:
There are actually no changes to this file other than the removal of the `pending` block, and the corresponding changes in indentation. (These specs were supposed to have been re-enabled after the Rails 5 upgrade, but that apparently never happened.)